classes/ImageManager.php

```diff
@@ -609,15 +609,14 @@ public static function create($type, $filename)
      *
      * @param int $width
      * @param int $height
-     *
-     * @return resource
```

Comment:
```php
      * @return resource|GdImage
```

Isn't it the new type returned in PHP8? Although I know you don't have a choice for the ignore comment But still this return type should be more accurate no?